src/pages/internal-news.tsx

```diff
@@ -0,0 +1,76 @@
+import { InferGetServerSidePropsType } from 'next'
+import {
+  BreadcrumbBar,
+  Breadcrumb,
+  BreadcrumbLink,
+} from '@trussworks/react-uswds'
+
+import { client } from '../lib/keystoneClient'
+
+import type { ArticleListItemRecord } from 'types'
+import Loader from 'components/Loader/Loader'
+import { useUser } from 'hooks/useUser'
+import { withPageLayout } from 'layout/DefaultLayout/PageLayout'
+import NavLink, { NavLinkProps } from 'components/util/NavLink/NavLink'
+import { GET_INTERNAL_NEWS_ARTICLES } from 'operations/cms/queries/getInternalNewsArticles'
+import { ArticleList } from 'components/ArticleList/ArticleList'
+import styles from 'styles/pages/news.module.scss'
+
+const InternalNews = ({
```

Comment:
Yeah I didn't add a test file for this because I didn't know details I know now. This page will become `/news` and I am probably going to have to update a lot of tests in the story for that, so I figured I could just add tests for it then.